### PR TITLE
Auto focus v2

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
## Status
In Progress

## Related Issues
fixes: https://github.com/demisto/etc/issues/16833
fixes: https://github.com/demisto/etc/issues/16469

## Description
AutoFocus v2 integration written python and formatted in the new package structure 

## Must have
- [x] Tests
- [x] Documentation https://github.com/demisto/etc/issues/17598
- [x] Code Review

### CODE REVIEW

**Comment by ShahafBenYakir** [comment]
@kirbles19 
[yml](https://github.com/demisto/content/pull/3736/files#diff-9e33901445219be3b018687739714d8f)
[md](https://github.com/demisto/content/pull/3736/files#diff-6e3175c729780fe5b6c5cfabeecb2f24)

File: Integrations/AutofocusV2/AutofocusV2.yml [hunk 1]
```diff
@@ -0,0 +1,410 @@
+commonfields:
+  id: AutoFocus V2
+  version: -1
+name: AutoFocus V2
+display: Palo Alto AutoFocus V2
```
Comment (bakatzir) [inline Integrations/AutofocusV2/AutofocusV2.yml]: Palo Alto Networks AutoFocus V2

**Comment by kirbles19** [comment]
.yml reviewed (nothing to review in the .md file). Need to improve some output descriptions.

**Comment by kirbles19** [comment]
@ShahafBenYakir 
[AutoFocus_DetailedDescription.txt](https://github.com/demisto/content/files/3310274/AutoFocus_DetailedDescription.txt)

**Comment by kirbles19** [comment]
Finished reviewing files.

**Comment by ShahafBenYakir** [comment]
- [ ] Should from the version of the integration (both in `conf.json` and in the `yml`)
- [ ] Remove Autofocus (v1) from skipped

File: Integrations/AutofocusV2/AutofocusV2.py [hunk 2]
```diff
@@ -0,0 +1,713 @@
...
+                'ContentsFormat': formats['text'],
+                'Contents': content,
+                'HumanReadable': hr
+            })
+
+
+''' COMMANDS'''
+
+
+def test_module():
```
Comment (yuvalbenshalom) [inline Integrations/AutofocusV2/AutofocusV2.py]: need to check that wrong host and bad auth are creating readable errors

File: Integrations/AutofocusV2/AutofocusV2.py [hunk 3]
```diff
@@ -0,0 +1,713 @@
...
+        'EntryContext': {'AutoFocus.Tag(val.ID == obj.ID)': context},
+        'HumanReadable': md
+    })
+
+
+def top_tags_search_command():
+    args = demisto.args()
+    scope = args.get('scope')
+    tag_class = args.get('class')
+    private = True if args.get('private') == 'True' else False
```
Comment (yuvalbenshalom) [inline Integrations/AutofocusV2/AutofocusV2.py]: you only need to write `args.get('private') == 'True'`
the same with the 3 following lines.

File: Integrations/AutofocusV2/AutofocusV2.py [hunk 4]
```diff
@@ -0,0 +1,713 @@
...
+requests.packages.urllib3.disable_warnings()
+
+''' GLOBALS/PARAMS '''
+PARAMS = demisto.params()
+API_KEY = PARAMS.get('api_key')
+# Remove trailing slash to prevent wrong URL path to service
+SERVER = 'https://autofocus.paloaltonetworks.com'
+# Should we use SSL
+USE_SSL = not PARAMS.get('insecure', False)
+PROXY = PARAMS.get('proxy')
```
[the remark this answers is not in the document]
Comment (ShahafBenYakir) [reply Integrations/AutofocusV2/AutofocusV2.py]: - [x] Check

File: Integrations/AutofocusV2/AutofocusV2.py (hunk 3 above)
Comment (ShahafBenYakir) [reply Integrations/AutofocusV2/AutofocusV2.py]: - [x] Check

File: Integrations/AutofocusV2/AutofocusV2.yml [hunk 5]
```diff
@@ -0,0 +1,406 @@
...
+  type: 8
+  required: false
+- display: Use system proxy
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+script:
+  script: '-'
+  type: python
```
[the remark this answers is not in the document]
Comment (ShahafBenYakir) [reply Integrations/AutofocusV2/AutofocusV2.yml]: - [x] Check

File: Integrations/AutofocusV2/AutofocusV2.py (hunk 2 above)
Comment (ShahafBenYakir) [reply Integrations/AutofocusV2/AutofocusV2.py]: There is no host parameter at all and the API key is already being checked. 
The integration is in Demo2 you can play with the test module.

**Pull request merged**